pyproject.toml

```diff
@@ -34,7 +35,7 @@ ignore = [
 ]
 select = ["ALL"]
 src = ["tap_linear"]
-target-version = "py311"
+target-version = "py38"
```

Comment:
why Python 3.8?

Reply:
Because ruff will otherwise propose code improvements that are not compatible with older yet supported Python versions. Notably https://github.com/leukeleu/tap-linear/pull/1/files/3bc74435bb0edf42e6c9193cb2c5634553808231#diff-938de46e643ab091cff6a7c23e4e752e8907e2266f47889d988923352f7a1058L10 was ruff's idea, but it broke on Python < 3.11.